src/app/service/map/_components/bottom-sheet/BottomSheetHeader.tsx

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable import/order */
+import Image from 'next/image';
+import useCss from '@/hooks/common/useCss';
+import BookmarkBorderIcon from '@mui/icons-material/BookmarkBorder';
+import BookmarkIcon from '@mui/icons-material/Bookmark';
+import OneSiteUrl from './OneSiteUrl';
+import { Phone, Star, StarHalf, ChevronRight } from 'lucide-react';
+import SquarePen from 'public/square-pen.svg';
+
+interface BottomSheetHeaderProps {
+  name: string;
+  address: string;
+  instagram_link?: string;
+  is_like?: boolean;
+  gym_type?: string;
+  phone_number: string;
+}
+
+function BottomSheetHeader({
+  name,
+  address,
+  instagram_link,
+  is_like,
+  gym_type,
+  phone_number,
+}: BottomSheetHeaderProps) {
+  useCss('https://unpkg.com/react-spring-bottom-sheet/dist/style.css');
+
+  const onClickBookmark = () => {
+    console.log('====북마크 클릭, 클릭에따라 api보내야함');
+  };
+
+  const addressSplit = address.split(' ');
+
+  return (
+    <>
+      {/* 인스타 및 북마크 */}
+      <section className="flex justify-end">
+        {instagram_link && (
+          <OneSiteUrl label="instagram" url={instagram_link} />
+        )}
+        <button type="button" className="ml-2" onClick={onClickBookmark}>
+          {is_like ? <BookmarkIcon /> : <BookmarkBorderIcon />}
+        </button>
+      </section>
+      {/* 클라이밍 이름 */}
+      <section>
+        <h1 className="flex items-left text-xl justify-left font-bold text-gray-800">
+          {name}
+        </h1>
+      </section>
+      {/* gym_type & 간략주소 */}
+      <section className="text-gray-500 font-normal text-sm leading-5 text-left ">
+        <div>{`${gym_type} · ${addressSplit[0]} ${addressSplit[1]}`}</div>
+      </section>
+      {/* 5개 빈 별공간 */}
+      <section className="flex text-sm leading-5 font-semibold text-black">
+        <div className="relative">
+          <div className="flex gap-4px">
+            {Array.from({ length: 5 }, (_, index) => (
+              <Star key={index} fill="#C3C6CC" strokeWidth={0} />
+            ))}
+          </div>
+          {/* 별 점수에 따라 넣어야함 (로직 아직안함), 별점 점수도 바꿔야함 */}
+          <div className="flex gap-4px absolute top-0">
+            <Star fill="#FFCB29" strokeWidth={0} />
+            <Star fill="#FFCB29" strokeWidth={0} />
+            <StarHalf fill="#FFCB29" strokeWidth={0} />
+          </div>
+        </div>
+        별점 점수
+        <ChevronRight size={24} strokeWidth={1} />
+      </section>
+
+      {/* 버튼 */}
+      <section
+        className="flex justify-around h-[36px] mt-10"
+        style={{ border: '1px solid red' }}
+      >
```

Comment:
```suggestion
     <section className="flex justify-around mt-10 border border-red-500">
```

Tailwind 문법으로 통일하면 더 좋을 것 같습니다

Reply:
어 이거는 나중에 다 되면 저거만 삭제하려고 넣어뒀어용ㅎ